### PR TITLE
UIAccess field in advanced token properties

### PR DESCRIPTION
**Pull request by diversenok**
I've added a new field to Token's properties tab that displays whether UIAccess flag for the token is enabled. This is a very interesting flag since it allows the process to bypass some UIPI restrictions.

### CODE REVIEW

File: phlib/include/phnativeinl.h [hunk 1]
```diff
@@ -1183,6 +1183,40 @@ PhGetTokenIsVirtualizationEnabled(
...
+FORCEINLINE
+NTSTATUS
+PhGetTokenIsUIAccessEnabled(
+    _In_ HANDLE TokenHandle,
+    _Out_ PBOOLEAN IsUIAccessEnabled
+    )
+{
+    NTSTATUS status;
+    ULONG returnLength;
+    DWORD uiAccess;
```
Comment (dmex) [inline phlib/include/phnativeinl.h]: This needs to be ULONG 😄

File: phlib/include/phnativeinl.h [hunk 2]
```diff
@@ -1183,6 +1183,40 @@ PhGetTokenIsVirtualizationEnabled(
...
+
+    status = NtQueryInformationToken(
+        TokenHandle,
+        TokenUIAccess,
+        &uiAccess,
+        sizeof(DWORD),
```
Comment (dmex) [inline phlib/include/phnativeinl.h]: Same here

File: phlib/include/phnativeinl.h (hunk 1 above)
Comment (diversenok) [reply phlib/include/phnativeinl.h]: Does it? ([1](https://msdn.microsoft.com/en-us/library/windows/desktop/aa379626(v=vs.85).aspx#TokenUIAccess), [2](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/content/ntifs/ne-ntifs-_token_information_class))

Well, there's not much difference, so if you insist... ¯\\\_(ツ)\_/¯

Comment (dmex) [reply phlib/include/phnativeinl.h]: NT uses ULONG for unsigned long types
Win32 uses DWORD for unsigned long types

These are NT kernel types and the wdk headers for those structures are using ULONG... Whomever wrote those MSDN documents is someone from the Win32 team using the incorrect terminology.

Comment (diversenok) [reply phlib/include/phnativeinl.h]: Ok, I'll take that into account. :+1:

**Comment by dmex** [comment]
Thanks! I usually fixup type/naming stuff without saying anything but I'm working on some other stuff :)

**Pull request merged**